src/state/InitState.ts

```diff
@@ -46,6 +46,8 @@ export class InitState implements IState{
     private dockerService: DockerService;
 
     private stateName: string;
+
+    private portsToCheck: number[] = [5551, 8545, 5600, 5433, 7546];
```

Comment:
Maybe we want to extend this to include all ports that are used by the local node (for example 8080 for the explorer)

Reply:
7546 will be removed probably, since it is not essential for the start of the local-node
At least thats i what we discussed with @georgi-l95 
Otherwise, I can add 8080 and the rest